.github/workflows/reanimated-typescript-compatibility-test-nightly.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         # TODO: Fetch versions from common source.
-        react-native-version: ['0.72', '0.73', '0.74', '0.75.0-rc.6', nightly]
+        react-native-version: ['0.72', '0.73', '0.74', '0.75.0', nightly]
```

Comment:
Why not 0.75.1 in here? Or straight up 0.75 and let it choose the latest

Reply:
You are right, it should've been 0.75